src/components/containers/ProfileSettings/ProfileSettings.jsx

```diff
@@ -0,0 +1,180 @@
+/* eslint-disable no-console */
+/* eslint-disable react/prop-types */
+import React, { Component } from 'react';
+import { Form } from 'semantic-ui-react';
+import { connect } from 'react-redux';
+
+import Button from '../../presentationals/Button/Button.jsx';
+import { generateFormData, toastbar } from '../../../utils/helpers';
+import {
+  updateUserProfileRequest,
+  uploadImageToServer,
+} from '../../../store/modules/Profile';
+import Loader from '../../presentationals/Loader/Loader.jsx';
+
+class Settings extends Component {
```

Comment:
Let's rename this component to `ProfileSetting`

Reply:
I have renamed it.